karma.conf.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function(config) {
```

Comment:
Is it really necessary to test this module in Chrome Headless? Seems like some pretty heavy machinery to bring in.

Reply:
Hm - maybe not. This module does need to run i. the browser, so on the basis it’s used that way by Unexpected I thought it made sense to cover that, but happy to go with this.

Reply:
My gut feeling says it's sufficient to cover that via the Unexpected test suite. Magicpen itself does not really integrate directly with browser apis.

The main reason I'm a bit hesitant to put browser tests into every module is that they tend to be fragile and add 100 MB worth of dev deps.

Reply:
@papandrou that’s a good point - it doesn’t really use any of the native facilities. How would you feel if I made it run the browser build on node as more of a smoke test that it built properly?

Reply:
Sure :+1: